docs/Operation-Architecture/Metadata_Search/Metadata_Search.rst

```diff
@@ -11,6 +11,16 @@ Zenko metadata searches can be performed from the command line or from
 the Orbit graphical user interface. Both searches use a common SQL-like
 syntax to drive a MongoDB search.
 
+.. toctree::
+   :maxdepth: 1
```

Comment:
as above. the Toctree should be in the top-level index page and carried over on the navigation. Putting it inside documents like this will create maintenance issues down the road (more places to keep in mind for broken links).

Reply:
Hidden (for now)